rasa/train.py

```diff
@@ -234,7 +247,13 @@ def train_nlu(
 
     config = get_valid_config(config, CONFIG_MANDATORY_KEYS_NLU)
 
-    nlu_data_directory = data.get_nlu_directory(nlu_data)
+    if not train_path:
+        # Training is not part of the stack model training, hence we have to load the
```

Comment:
I don't get this comment

Reply:
tried to improve it, does it make more sense now?